routes/buzzwords.js

```diff
@@ -0,0 +1,176 @@
+const express = require('express');
+const router = express.Router();
+const routerReset = express.Router();
+const routerHeard = express.Router();
+
+let buzzwordObj = { buzzwords: [] };
+let scoreBoard = { totalScore: 0 };
+
+function checkKeys(bodyObject) {
+  if (bodyObject.hasOwnProperty('buzzWord') && bodyObject.hasOwnProperty('points')) {
+    return true;
+  }
+
+  return false;
+}
+
+function checkBodyObjLength() {
+  let buzzWordArray = buzzwordObj.buzzwords;
+
+  if (buzzWordArray.length === 5) {
+    return false;
+  }
+  return;
+}
+
+function createBuzzword(buzzword, points) {
+  let newBuzzword = {};
+  newBuzzword.buzzWord = buzzword;
+  newBuzzword.points = points;
+
+  let buzzWordArray = buzzwordObj.buzzwords;
+  buzzWordArray.push(newBuzzword);
+
+  return;
+}
+
+function checkBuzzwordExists(buzzword) {
+  let buzzWordArray = buzzwordObj.buzzwords;
+
+  function hasBuzzword(object) {
+    return object.buzzWord === buzzword;
+  }
+
+  return buzzWordArray.some(hasBuzzword);
+}
+
+function updateBuzzWordPoints(buzzword, points) {
+  let buzzWordArray = buzzwordObj.buzzwords;
+
+  buzzWordArray.forEach((objectPair) => {
+    if (objectPair.buzzWord === buzzword) {
+      objectPair.points = points;
+    }
+    return;
+  });
+  return;
+}
+
+function deleteBuzzWord(buzzword) {
+  let buzzWordArray = buzzwordObj.buzzwords;
+  let deleteIndex = 0;
+
+  buzzWordArray.forEach((objectPair, index) => {
+    if (objectPair.buzzWord === buzzword) {
+      deleteIndex = index;
+    }
+    return;
+  });
+
+  buzzWordArray.splice(deleteIndex, 1);
+  return;
+}
+
+function resetServer() {
+  buzzwordObj = { buzzwords: [] };
+  scoreBoard = { totalScore: 0 };
+  return;
+}
+
+function updateScore(buzzword) {
+  let buzzWordArray = buzzwordObj.buzzwords;
+  let score = 0;
+
+  buzzWordArray.forEach((objectPair, index) => {
+    if (objectPair.buzzWord === buzzword) {
+      score = objectPair.points;
+    }
+    return;
+  });
+
+  scoreBoard.totalScore = Number(scoreBoard.totalScore) + Number(score);
+  return;
+}
+
+router
+  .route('/')
+  .get((req, res) => {
+    res.send(JSON.stringify(buzzwordObj));
+    return;
+  })
+  .post((req, res) => {
+    if (!checkKeys(req.body)) {
+      res.send(`{ "success": false }`);
+      return;
+    }
+
+    if (checkBuzzwordExists(req.body.buzzWord)) {
+      console.log('BuzzWord already exists!');
+      res.send(`{ "success": false }`);
+      return;
+    }
+
+    if (checkBodyObjLength === false) {
```

Comment:
This is a function, but you're not invoking it. I think this would always be false. Function !== boolean.